src/libraries/Common/src/Interop/Windows/Netapi32/Interop.DsRoleFreeMemory.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Netapi32
+    {
+        [LibraryImport(Libraries.Netapi32)]
+        internal static partial int DsRoleFreeMemory(IntPtr buffer);
```

Comment:
```suggestion
        internal static partial void DsRoleFreeMemory(IntPtr buffer);
```

https://learn.microsoft.com/windows/win32/api/dsrole/nf-dsrole-dsrolefreememory